fluent-package/msi/source.wxs

```diff
@@ -196,7 +196,7 @@
     <Property Id="CreateCompatTdAgentBat" Value=" "/>
     <CustomAction Id="SetCreateCompatTdAgentBat"
                   Property="CreateCompatTdAgentBat"
-                  Value="&quot;cmd&quot; /c &quot;fsutil hardlink create [FLUENTPROJECTLOCATION]bin\td-agent.bat [FLUENTPROJECTLOCATION]fluentd.bat&quot;"/>
+                  Value="&quot;cmd&quot; /c &quot;fsutil hardlink create ^&quot;[FLUENTPROJECTLOCATION]bin\td-agent.bat^&quot; ^&quot;[FLUENTPROJECTLOCATION]fluentd.bat^&quot;&quot;"/>
```

Comment:
Thanks for this fix!

I have a question.
What is the difference between `'` and `^&quot;`?
I understand what `^&quot;` means, but can't we use `'` here?

Reply:
Oh, sorry. I understand.
In Command Prompt, we can't use `'` for strings...

Reply:
`fluentdopt` is a String to be passed to Ruby.exe, so we can use `'`.
On the other hand, for the options to be handled in Command Prompt itself, we have to use espaped `"`.